src/entity/user/User.ts

```diff
@@ -122,6 +122,9 @@ export class User {
   @Column({ type: 'text', nullable: true })
   readmeHtml?: string;
 
+  @Column({ type: 'text', nullable: true })
+  acquisitionChannel: string;
```

Comment:
From RFC this should accept enum values eg. `AcquisitionChannel` from https://github.com/dailydotdev/daily-api/pull/1716